recurly.services.yml

```diff
@@ -3,42 +3,42 @@ services:
     class: Drupal\recurly\Access\RecurlyAccessLocalAccount
     tags:
       - { name: access_check, applies_to: _access_check_recurly_local_account }
-    arguments: ['@current_route_match']
+    arguments: ['@current_route_match', '@recurly.config_manager']
```

Comment:
Interesting/annoying that you have to declare this here and in the `create()` method.